packages/components/src/button/README.md

```diff
@@ -7,6 +7,7 @@ Buttons express what action will occur when the user clicks or taps it. Buttons
 ```jsx
 import Gridicon from 'components/gridicons';
 import { Button } from '@automattic/components';
+import '@automattic/calypso-color-schemes/dist/calypso-color-schemes.css';
```

Comment:
Importing the color schemes in `Button` so the consumer doesn't have to do it sounds like a great idea which I was so close to start coding!

I just realised unfortunately that since the `@import` defines a bunch of global CSS variables we only want to `@import` it once in the entire application. If we `@import` it in `Button`, the consumer cannot import it anywhere. That gets more complicated when you consider that other components require that `@import` and the possibility for multiple versions of the `@automattic/calypso-color-schemes` package to be used within the same app. 

<details>
<summary>🤔</summary>
<br/>
:trollface: A CSS-in-JS theme provider would nicely solve these pain points
</details>

Can you think of a better way to document in `@automattic/components` that `@automattic/calypso-color-schemes/dist/calypso-color-schemes.css` needs to manually be imported once per application?

Reply:
Ah theming 😭

I think package level documentation is needed in general (not this PR though) since we require some things from the client e.g. supported React versions, a sass-loader is required, etc. And I think just add to that list that clients must include this magic file once in their app.

But alas I think you're right, the constants just need to be defined once per app. I was hoping that tree-shaking magic would be able to help us with this.

Reply:
I've removed the import from the `Button` readme and added it in a `@automattic/components` readme.

Reply:
>To solve this, can you think of a better way to document in @automattic/components that @automattic/calypso-color-schemes/dist/calypso-color-schemes.css needs to manually be imported once per application?

`@automattic/calypso-color-schemes` is problematic for re-use because it contains very WordPress.com specific CSS vars; so it's not actually "framework package" like the name suggests. Things like "sidebar background-color" etc.

Those then end up in each bundle where we need a mere button, e.g. in Jetpack.

`@automattic/color-studio` seems better that way but still at on of CSS variables that end up not being used.

Reply:
👍 Yep, I agree, long term the variable naming isn't amazing, in some places its overly generic and in others its overly specific but its currently a dependency of the components.

Longer term I'd like to see a minimal set of variables that can be customised by the consumer - similar to Material's idea of surfaces. @davewhitley and I were [experimenting](https://salad-ui-playground.now.sh/plugins/featured) with these [ideas](https://salad-ui.now.sh/foundations/color/) (click the icon in the bottom right to toggle the variables).

For now, asking the consumer to import the variables once universally in the app feels like an acceptable workaround until we get around to deciding on and refactoring those things.